docs/data-sources/connector.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "apono_connector Data Source - terraform-provider-apono"
+subcategory: ""
+description: |-
+  Use this data source to get the ID of an Apono connector for when creating integrations.
+---
+
+# apono_connector (Data Source)
+
+Use this data source to get the ID of an Apono connector for when creating integrations.
+
+## Example Usage
+
+```terraform
+data "apono_connector" "prod_us" {
```

Comment:
How it's connected to our module?
we need to be able to take the ide from the tf module that we give the account for install the connector